.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+python:
+#  - 3.5
+  - 3.6
+  - 3.7
+  - nightly
```

Comment:
I would add 3.8 to the matrix. Nightly could be designated that failures are ok. https://docs.travis-ci.com/user/build-matrix/#rows-that-are-allowed-to-fail

This allows the green checkmark to be given to the whole run. Yet Travis will report that nightly failed so use still know how it is building.